deploy error for todoapp is not explicit

I'm trying to deploy the todoapp from my dagger fork. The error is not explicit.

`github actions log for todoapp`
```
4:02PM INF actions.build.run._exec | #20 12.08
4:02PM INF actions.build.run._exec | #20 12.08 The build folder is ready to be deployed.
4:02PM INF actions.build.run._exec | #20 12.08
4:02PM INF actions.build.run._exec | #20 12.08 Find out more about deployment here:
4:02PM INF actions.build.run._exec | #20 12.08
4:02PM INF actions.build.run._exec | #20 12.08   bit.ly/CRA-deploy
4:02PM INF actions.build.run._exec | #20 12.08
4:02PM INF actions.build.run._exec | #20 12.12 Done in 11.76s.
4:02PM INF actions.deploy.container._exec | #22 0.845 curl: (22) The requested URL returned error: 404
4:02PM INF actions.deploy.container._exec | #22 0.850 {"code":404,"message":"Not Found"}
4:02PM ERR actions.deploy.container._exec | failed: process "bash --norc -e -o pipefail /bash/scripts/deploy.sh" did not complete successfully: exit code: 1
    duration=900ms
4:02PM FTL system | failed to execute plan: task failed: actions.deploy.container._exec: process "bash --norc -e -o pipefail /bash/scripts/deploy.sh" did not complete successfully: exit code: 1


Error: Process completed with exit code 1.
```

--- a/pkg/universe.dagger.io/netlify/deploy.sh
+++ b/pkg/universe.dagger.io/netlify/deploy.sh
@@ -15,6 +15,7 @@ create_site() {
 
 	# shellcheck disable=SC2181
 	if [ $? -ne 0 ]; then
+		>&2 echo "Error creating site [${NETLIFY_SITE_NAME}] for account [${NETLIFY_ACCOUNT}]"
 		cat body >&2
 		exit 1
 	fi
@@ -47,7 +48,7 @@ netlify link --id "$site_id"
 netlify deploy \
 	--build \
 	--site="$site_id" \
-	--prodIfUnlocked |
+	--prod |
 	tee /tmp/stdout
 
 url="$(grep </tmp/stdout Website | grep -Eo 'https://[^ >]+' | head -1)"

